site/content/api/Event-API.md

```diff
@@ -76,4 +81,63 @@ The following sequence diagram illustrates the flow of messages involved in a *B
 
 **Message Format**
 
+**Arbitrary events from the device**
```

Comment:
IMHO we should omit this line. The message format is still defined as described by the Telemetry API regardless of whether a specific *well-known* event is sent. The *well-known* event types simply define some particular values that the properties need to be set to.